packages/web/docs/src/components/trusted-by-section.tsx

```diff
@@ -0,0 +1,19 @@
+import { cn } from '@theguild/components';
+import { AligentLogo, KarrotLogo, LinktreeLogo, MeetupLogo, SoundYXZLogo } from './company-logos';
+
+export function TrustedBySection(props: React.HTMLAttributes<HTMLElement>) {
```

Comment:
I moved it out to a separate file because it's included both in the landing page and in the Case Studies index page design.